docs/changelog.rst

```diff
@@ -41,7 +41,12 @@ And there are a few bug fixes too:
 The last release, 1.3.19, also erroneously reported its version as "1.3.18"
 when you typed ``beet version``. This has been corrected.
 
+????
+
```

Comment:
Under "Fixes" is probably fine for this sort of thing. And we should probably note that the dependency is for the `thumbnails` plugin, not a global dependency for beets.